mempool/mempool.go

```diff
@@ -85,7 +88,7 @@ type ByFeeRate struct {
 	TicketsDetails
 }
 
-// Less returns TicketsDetails[i].FeeRate < TicketsDetails[j].FeeRate
+// Less compares fee rates by rate_i < ate_j
```

Comment:
`ate_j` -> `rate_j`

Reply:
oh, it would be good to stake if this implementation provides ascending or descending ordering of the sorted elements